src/logging/Manager.cc

```diff
@@ -739,7 +755,7 @@ bool Manager::Write(EnumVal* id, RecordVal* columns_arg)
 			auto v = filter->policy->Invoke(columns,
 							IntrusivePtr{NewRef{}, id},
 							IntrusivePtr{NewRef{}, filter->fval});
-			if ( v  && ! v->AsBool() )
+			if ( stream_veto || (v  && ! v->AsBool()) )
```

Comment:
Should the check for `stream_veto` be outside of this `if`? Right now, if there's not filter policy hook, wouldn't the global veto be ignored?

Reply:
Good catch, yep. I forgot that there may be no filters at all.

Reply:
I added a testcase for that situation, while I was at it.